rust/src/delta.rs

```diff
@@ -1731,29 +1811,29 @@ mod tests {
                 Action::metaData(action) => {
                     assert_eq!(DeltaTableMetaData::try_from(action).unwrap(), delta_md);
                 }
-                Action::commitInfo(action) => {
-                    let mut modified_action = action;
-                    let timestamp = serde_json::Number::from(0i64);
-                    modified_action["timestamp"] = Value::Number(serde_json::Number::from(0i64));
-                    let mut expected = Map::<String, Value>::new();
-                    expected.insert(
-                        "operation".to_string(),
-                        serde_json::Value::String("CREATE TABLE".to_string()),
-                    );
-                    expected.insert(
-                        "userName".to_string(),
-                        serde_json::Value::String("test user".to_string()),
-                    );
-                    expected.insert(
-                        "delta-rs".to_string(),
-                        serde_json::Value::String(crate_version().to_string()),
-                    );
-                    expected.insert(
-                        "timestamp".to_string(),
-                        serde_json::Value::Number(timestamp),
-                    );
-                    assert_eq!(modified_action, expected)
-                }
+                // Action::commitInfo(action) => {
```

Comment:
Currently that test fails since in this PR we now create basic commit info from the `DeltaOperation` enum. If we stick with that, the test needs to be refactored and maybe moved.